docs/synapse/userguides/ug015_storm_ref_pivot.rst

```diff
@@ -220,7 +221,7 @@ N/A
 * The ``limit=`` parameter can be provided as input to the ``refs()`` operator itself when using Operator syntax. Alternately the ``limit()`` operator_ can be used after the ``refs()`` operator (in either Operator or Macro syntax) to specify a limit on the number of nodes returned.
```

Comment:
Suggest writing the refs() "Usage Notes" to parallel the join() Usage Notes, e.g., start with:
* refs() does not consume nodes, so the results of a refs() operation will include both the original working set as well as the resulting set of nodes.
... stuff ...
* Because refs() does not consume nodes, this impacts the results returned by either the limit= parameter of the limit() operator.
... stuff ...

Reply:
fixed